go.mod

```diff
@@ -3,12 +3,19 @@ module sigs.k8s.io/kubebuilder/v3
 go 1.15
 
 require (
+	github.com/cloudflare/cfssl v1.5.0
 	github.com/gobuffalo/flect v0.2.2
+	github.com/markbates/pkger v0.17.1
 	github.com/onsi/ginkgo v1.15.0
 	github.com/onsi/gomega v1.10.5
 	github.com/spf13/afero v1.2.2
-	github.com/spf13/cobra v0.0.7
+	github.com/spf13/cobra v1.1.1
```

Comment:
Do you need to update cobra for this PR?

I definetely think that bumping cobra is somethign we need to do, but if it is not needed forn this PR we should probably do it in a separate PR.

Reply:
No, this is just what happened with I ran `go test ; go mod tidy`

Reply:
Checking if it works with v0.0.7 and if `go mod tidy` doesn't bump it afterwards would confirm if this is needed or not.

Reply:
Running either `go build ./pkg/...` or `go mod tidy` bumps this to v1.0.0 or higher.

Reply:
I think this is ok.